ofl/imfelldwpicasc/DESCRIPTION.en_us.html

```diff
@@ -1 +1,21 @@
\ No newline at end of file
+<p>
+IM Fell DW Pica SC (small caps) is one family within a revival project of all the Fell Types – a collection of types bequeathed by John Fell to the University of Oxford in 1686, who was Bishop of Oxford and Dean of Christ Church. 
+The fonts are all characterized by an unrefined look, where letterforms are intentionally uneven, which can be attributed to the strong fidelity of this particular revival process.
+</p>
+<p>
+All the typefaces are useful today for websites, books, essays, magazines and advertisements, and they come in different variations designed for different sizes of use. 
+IM Fell DW Pica SC is intended to be used at 12.5 points, suitable for body text.
```

Comment:
Small cap fonts are not suitable for body text.

Reply:
Perhaps you could say "IM Fell DW Pica SC can be used to improve the appearance of acronyms in body text."